src/model/Model_Currency.cpp

```diff
@@ -175,8 +175,9 @@ wxString Model_Currency::toString(double value, const Data* currency, int precis
 
 const wxString Model_Currency::fromString2Default(const wxString &s, const Data* currency)
 {
-    wxString str = s;
-    str.Trim();
+    // Remove any characters before the number stars. Useful especially for removing currency symbols.
+    wxString str = s.Mid(s.find_first_of("-0123456789"));
```

Comment:
Personally, I feel that it's incorrect. It broke an imput like .20
I beleave that better to use regex
http://stackoverflow.com/questions/1357946/regexp-to-parse-currency-value
https://bytes.com/topic/c-sharp/answers/508843-regular-expression-parse-currency-any-symbol